README.adoc

```diff
@@ -54,6 +54,16 @@ teardown()
 
 ```
 
+== Environment Variables
+|===
+| Name | Purpose
+| TEST_FUNCTION | Bats variable the is used to control debugging i shellmock. When
+set to the name of a test case then all tmp and debug output is
+preserved.
```

Comment:
The idea was to enable debugging in conjunction with TEST_FUNCTION in bats.  My rationale was if you are trying to isolate and run a single test then you are likely debugging.  If you are not using skipIfNot in your bats file then I can see that tests would run and debug would happen.

If configured as described in the read me with skipIfNot then I would expect if you set TEST_FUNCTION to foo then no tests would run.

I could make them mutually exclusive.  It is this way primarily because this is how I was doing test debugging. 

If there is an need to separate them lets raise a separate issue and I can take a look, or you could propose something in a PR if you like.

Reply:
No that's fine by me ! I'm actually satisfied with the way it works and it seems I have misread the documentation